### PR TITLE
Fix #49, tox can be used by user

### LINKED ISSUE #49: Check tox.
Make sure a run using tox will not fail.
I guess it is better to do nothing than to have a horrible error message.

### PR DESCRIPTION
**Pull request by uggla**
- Allow user to use tox
- Docker is required !

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ coverage>=3.6
 mock>=1.0.1
 docker-py>=1.1.0
 path.py>=5.2
+testrepository>=0.0.20
```
Comment (bcornec) [inline test-requirements.txt:10]: Each time there is a modification here, we need to align the deps for packages. At least create a ticket so I know and can do it.

**Comment by bcornec** [comment]
Not sure why you mention docker is required, as that PR doesn't explicitely mention docker anywhere ?

File: test-requirements.txt (hunk 1 above)
Comment (uggla) [inline test-requirements.txt:10]: Sorry, I didn't do it because it is test deps and we said that we currently don't manage test deps.

**Comment by uggla** [comment]
You refused the PR...   :crying_cat_face:
Re opening it...

**Comment by bcornec** [comment]
Sorry, bad manipulation...

**Pull request merged**